i18n/locales/en.ts

```diff
@@ -961,11 +961,11 @@ const translations = {
         'This is the raw transaction with each component labeled:',
       success_message_line_two: 'version:',
       success_message_line_three: 'number of inputs:',
-      success_message_line_four: 'hash of tx being spent by input #0:',
+      success_message_line_four: 'hash of the tx that input #0 came from:',
       success_message_line_five:
-        'index of output of tx being spent by input #0:',
+        'index of input #0 in the tx that created it:',
```

Comment:
Still don't love this wording, but I think it's an improvement. Open to suggestions here.

Reply:
`index of input #0 from the funding tx` ?